tests/test_generators.py

```diff
@@ -20,8 +22,97 @@ def test_smoke(self, model_name):
         desired_result_count = 2
 
         results = target(
-            prompts,
-            "",
+            prompts=prompts,
+            topic="",
+            mode="tests",
+            topic_description="",
+            num_samples=desired_result_count,
+        )
+        assert results is not None
+        assert len(results) == desired_result_count
+        for item in results:
+            assert isinstance(item, str)
+
+    @pytest.mark.parametrize("model_name", TRANSFORMER_PIPELINE_MODELS)
+    def test_with_topics(self, model_name):
+        hf_model = pipeline("text-generation", model=model_name)
+        target = generators.Transformers(hf_model.model, hf_model.tokenizer)
+
+        prompts = [
+            ("id A", "some string", "Great hotel"),
+            ("id B", "some_string", "Bathroom too small"),
+        ]
+
+        desired_result_count = 2
+
+        results = target(
+            prompts=prompts,
+            topic="",
+            mode="tests",
+            topic_description="",
+            num_samples=desired_result_count,
+        )
+        assert results is not None
+        assert len(results) == desired_result_count
+        for item in results:
+            assert isinstance(item, str)
+
+
+GENERATOR_PIPELINE_MODELS = [
+    "facebook/opt-125m",
+    "facebook/opt-350m",
+    "EleutherAI/gpt-neo-125M",
+    "gpt2",
+    "bigscience/bloom-560m",
```

Comment:
Nice set of models to test. Just want to make sure the GH actions machines we use will consistently be able to load and use these

Reply:
I doubt that the build cluster has GPUs, so these probably just all run on the CPU anyway?